context/spec/022-unified-design-system/design-system/project/ui_kits/app/tokens.css

```
@@ -0,0 +1,274 @@
+   retired in v2 to keep the system consistent and product-fit.
+
+   ── ACCENT ──────────────────────────────────────────────────────────────────
+   The brand accent is a single swappable variable. Set `data-accent` on <html>
+   to one of: indigo · sage · clay · plum. Default (no attribute) = sage.
+   Each preset supplies a LIGHT tone (mid-dark, for cream) and a DARK tone
+   (lighter, for charcoal). Hover/active and the "Playing" status derive from it.
+
+   Fonts load from Google Fonts (Geist, Geist Mono, Space Grotesk).
+   ========================================================================== */
+
+@import url("https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500;600;700&family=Geist:wght@400;500;600;700;800&family=Space+Grotesk:wght@400;500;600;700&display=swap");
```

Stylelint (17.12.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500;600;700&family=Geist:wght@400;500;600;700;800&family=Space+Grotesk:wght@400;500;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500;600;700&family=Geist:wght@400;500;600;700;800&family=Space+Grotesk:wght@400;500;600;700&display=swap"" (import-notation)

(import-notation)